src/main/scala/treadle/executable/ExecutionEngine.scala

```diff
@@ -153,6 +157,43 @@ class ExecutionEngine(
     }
   }
 
+  def randomize(): Unit = {
+    val rand = new Random
+
+//    val symbolsToDo: Seq[Symbol] = {
+//      symbolTable.registerNames.map(name => symbolTable(name)).toSeq ++
+//        symbolTable.moduleMemoryToMemorySymbol.values.flatten
+//    }
+    val symbolsToDo: Seq[Symbol] = symbolTable.symbols.toSeq
+
+    symbolsToDo.foreach { symbol =>
+      rand.setSeed(symbol.name.hashCode + userRandomSeed)
```

Comment:
Are you sure that reseeding the random number generator is a good idea in terms of getting "good" random results?
If you can assure that all values are generated in a deterministic order, e.g. by sorting all symbols by their name, they should be deterministic and only require the random number generated to be seeded a single time.

Reply:
It's a good question, I'll look into it